Userland/Libraries/LibCore/EventLoop.cpp

```diff
@@ -263,7 +263,7 @@ class InspectorServerConnection : public Object {
         }
 
         if (type == "SetInspectedObject") {
-            auto address = request.get_deprecated("address"sv).to_number<FlatPtr>();
+            auto address = request.get_addr("address"sv).value_or(0);
```

Comment:
Can this also be left as an Optional?
```c++
auto address = request.get_addr("address"sv);
```

If it's nullopt, the `==` comparisons below will happily fail